scripts/ganache-cli.sh

```diff
@@ -0,0 +1,42 @@
+#!/usr/bin/env bash
+
+# Exit script as soon as a command fails.
+set -o errexit
+
+# Executes cleanup function at script exit.
+trap cleanup EXIT
+
+cleanup() {
+  # Kill the ganache instance that we started (if we started one and if it's still running).
+  if [ -n "$rpc_pid" ] && ps -p $rpc_pid > /dev/null; then
+    kill -9 $rpc_pid
+  fi
+}
+
+setup_testing_variables() {
+  PORT=${PORT-8545}
+  BALANCE=${BALANCE-100000}
+  # Allow yuge gas limit for solidity tests
+  GAS_LIMIT=${GAS_LIMIT-100000000}
+  NETWORK_ID=${NETWORK_ID-15}
+  ACCOUNTS=${ACCOUNTS-200}
+}
+
+start_ganache() {
+  echo "Starting ganache-cli on byzantium..."
+  # Rollback to byzantium hardfork to more closely mirror 2017 hardfork state
```

Comment:
~Nit: the sale was done pre byzantium (unsure if ganache-cli supports homestead).~

Edit: just saw that trying to run it on an earlier version didn't work